src/Apps/Auction/__tests__/routes.test.ts

```diff
@@ -31,6 +31,12 @@ describe("Auction/routes", () => {
     me: data.me,
   })
 
+  it("renders the Auction FAQ view", async () => {
+    const { status } = await render("/auction-faq", mockResolver(Fixture))
```

Comment:
Can we add a couple more tests here?

Would be nice to verify toggle behavior, and also that entire sections don't show when content is missing.

Reply:
Added a test file with these tests.